mapswipe_workers/python_scripts/copy_user_contributions_in_firebase.py

```diff
@@ -0,0 +1,46 @@
+from mapswipe_workers.auth import firebaseDB
+from mapswipe_workers.definitions import logger
+
+
+def get_all_users():
+    """Get the user ids from all users in Firebase DB."""
+    fb_db = firebaseDB()
+    users = fb_db.reference(f"v2/users/").get(shallow=True)
+    uid_list = users.keys()
+    return uid_list
+
+
+def copy_user_contributions(uid):
+    """Copy user contributions in Firebase from `users` to `userContributions`."""
+    fb_db = firebaseDB()
+    try:
+        ref = fb_db.reference(f"v2/users/{uid}/contributions")
+        user_contributions = ref.get()
+        # only set username for users with display_name
```

Comment:
I don't understand this comment. Is it a leftover from a previous version?

Reply:
this comment should be removed. correct.